### PR TITLE
Add expose support from bundle drops

### PR DESCRIPTION
**Pull request by hatched**
When dropping bundles with exposed services it should expose the service.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

This one is a little interesting because we don't actually allow the user to expose a service until it has been committed. So when you drop a bundle the service is marked as exposed but that's not surfaced anywhere. This issue is compounded because we don't have the service icon exposed state completed yet (card was created)

- bundlelib and jujucorebundle have not yet been updated to support this so to QA you will need to go into the test/data directory and then drag the wordpress-bundle-recordset.json onto the canvas. It should import the bundle.
- Before committing run `app.db.services.each(function(s) { console.log(s.get('exposed')); });` and it should show that one is exposed.
- Commit
- Run the above code again and it should show one exposed
- Open the inspector for mysql and you should see Exposed On

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3492/
Test PASSed.

File: jujugui/static/gui/src/app/utils/bundle-importer.js [hunk 1]
```diff
@@ -339,6 +339,21 @@ YUI.add('bundle-importer', function(Y) {
...
     /**
+      Executes the expose method call.
+
+      @method _execute_expose
+      @param {Object} record The expose record to execute.
+      @param {Function} next The method to call to trigger the executor to
+        move on to the next record.
+    */
+    _execute_expose: function(record, next) {
+      // Grab the actual record Id
```
Comment (frankban) [inline jujugui/static/gui/src/app/utils/bundle-importer.js]: Please complete the sentence.

**Comment by frankban** [comment]
:+1: with only a doc trivial.
Thank you!

**Comment by kadams54** [comment]
Code is :+1: - working on the QA.

**Comment by kadams54** [comment]
QA is :ok:

**Comment by hatched** [comment]
Thanks for the reviews! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**